devices/Paexo/src/Paexo.cpp

```diff
@@ -346,10 +418,48 @@ bool Paexo::open(yarp::os::Searchable& config)
                                                                                                                                         pImpl->ftSensorPrefix + "Right" + pImpl->ftSensorName)};
 #endif
 
-    // Initialize wearable actuators
+    // Initialize wearable actuators for left and right motor
     pImpl->motorActuatorPrefix = getWearableName() + actuator::IMotor::getPrefix();
-    pImpl->paexoMotorActuator = ElementPtr<PaexoImpl::PaexoMotorActuator>{std::make_shared<PaexoImpl::PaexoMotorActuator>(pImpl.get(),
-                                                                                                                         pImpl->motorActuatorPrefix + pImpl->motorActuatorName)};
+    const std::string leftActuatorName = pImpl->motorActuatorPrefix + "Left" + pImpl->motorActuatorName;
+    pImpl->paexoLeftMotorActuator = ElementPtr<PaexoImpl::PaexoMotorActuator>{std::make_shared<PaexoImpl::PaexoMotorActuator>(pImpl.get(), leftActuatorName)};
+
+    const std::string rightActuatorName = pImpl->motorActuatorPrefix + "Right" + pImpl->motorActuatorName;
+    pImpl->paexoRightMotorActuator = ElementPtr<PaexoImpl::PaexoMotorActuator>{std::make_shared<PaexoImpl::PaexoMotorActuator>(pImpl.get(), rightActuatorName)};
+
+    // Initialize yarp control ports
+
+    // Check yarp network initialization
+    if (!yarp::os::Network::isNetworkInitialized()) {
+        yInfo() << LogPrefix << "Initializing yarp network";
+        yarp::os::Network();
+    }
+
+    yInfo() << LogPrefix << "Initiailizing PaexoMotorControlPort for " << leftActuatorName;
+
+    const std::string leftActatorName = pImpl->getValidYarpName(leftActuatorName);
+    pImpl->paexoLeftMotorControlPort = std::make_unique<PaexoImpl::PaexoMotorControlPort>(leftActuatorName, pImpl->paexoLeftMotorActuator);
+    pImpl->paexoLeftMotorControlPort->useCallback();
+
+
+    if (!pImpl->paexoLeftMotorControlPort->open(leftActatorName))
```

Comment:
`leftActatorName` -> `leftActuatorName`

Reply:
Done in https://github.com/robotology/wearables/pull/124/commits/47851b138e1739a7204181345094fbf4b718013f